src/tutorial/scheduling/datetime-cycling.rst

```diff
@@ -383,7 +388,7 @@ Putting It All Together
    subgraph cluster_T06 {
        label="+PT6H"
        style="dashed"
-       "forecast.t06" [label="forecast\n+PT6H"]
+       "forecast.t06" [label="+PT6H/forecast"]
```

Comment:
```suggestion
       "forecast.t06" [label="+PT6H\nforecast"]
```

Reply:
Yes, good catch, I thought I'd reverted all of these.